docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -63,6 +63,11 @@ notebooks:
     # A value of true will automatically display the latest tag updated as default.
     # A value of false will retain the default version that has been specified via the Dockstore UI.
     latestTagAsDefault: <Boolean>
+
+    # A boolean that will control whether DOIs are automatically issued for tags. Default: True.
```

Comment:
Setting it to `true` right now will cause the push to fail. The only acceptable value is `false` since we control the enabling via the admin endpoint

Reply:
Hmmmm, actually fail as in aborting or just fail to do anything?